bin/copyRawDataToPrm.sh

```diff
@@ -281,64 +281,72 @@ function splitSamplesheetPerProject() {
 		#  * either only demultiplexing was requested via the samplesheet
 		#  * or when disabled on the commandline by enabling "archiveMode".
 		#
+		log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "mergedSamplesheet = ${mergedSamplesheet}."
 		if [[ "${mergedSamplesheet}" == 'true' ]]
 		then
 			_project=$(echo "${_project}" | grep -Eo 'GS_[0-9]+')
-		fi
-		
-		if [[ "${archiveMode}" == 'false' ]]; then
-			#
-			# Skip project if demultiplexing only.
-			#
-			local _projectSampleSheet
-			_projectSampleSheet="${PRM_ROOT_DIR}/Samplesheets/${_project}.${SAMPLESHEET_EXT}"
-			head -1 "${_sampleSheet}" > "${_projectSampleSheet}.tmp"
-			grep "${_project}" "${_sampleSheet}" >> "${_projectSampleSheet}.tmp"
-			mv "${_projectSampleSheet}"{.tmp,}
-			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Created ${_projectSampleSheet}."
-			#
-			# Get pipeline/analysis values from samplesheet.
-			#
-			IFS="${SAMPLESHEET_SEP}" read -r -a _sampleSheetColumnNames <<< "$(head -1 "${_projectSampleSheet}")"
-			for (( _offset = 0 ; _offset < ${#_sampleSheetColumnNames[@]} ; _offset++ ))
-			do
-				_sampleSheetColumnOffsets["${_sampleSheetColumnNames[${_offset}]}"]="${_offset}"
-			done
-			if [[ -n "${_sampleSheetColumnOffsets["${PIPELINECOLUMN}"]+isset}" ]]; then
-				_pipelineFieldIndex=$((${_sampleSheetColumnOffsets["${PIPELINECOLUMN}"]} + 1))
-				_projectFieldIndex=$((${_sampleSheetColumnOffsets["${PROJECTCOLUMN}"]} + 1))
-				readarray -t _pipelines < <(tail -n +2 "${_projectSampleSheet}" | cut -d "${SAMPLESHEET_SEP}" -f "${_pipelineFieldIndex}" | sort | uniq )
-				if [[ "${#_pipelines[@]}" -lt '1' ]]
-				then
-					log4Bash 'ERROR' "${LINENO}" "${FUNCNAME:-main}" '0' "${_sampleSheet} does not contain at least one value in the ${PIPELINECOLUMN} column."
-					log4Bash 'ERROR' "${LINENO}" "${FUNCNAME:-main}" '0' "Skipping ${_run} due to error in samplesheet."
+			log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "project will now be ${_project}."
+			if ssh "${DATA_MANAGER}@${sourceServerFQDN}" "touch ${SCR_ROOT_DIR}/logs/${_project}/run01.rawDataCopiedToPrm.finished"
+			then
+				log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Succesfully created ${SCR_ROOT_DIR}/logs/${_project}/run01.rawDataCopiedToPrm.finished on ${sourceServerFQDN}"
+			else
+				log4Bash 'ERROR' "${LINENO}" "${FUNCNAME:-main}" '0' "Could not create ${SCR_ROOT_DIR}/logs/${_project}/run01.rawDataCopiedToPrm.finished on ${sourceServerFQDN}"
+				mv "${_controlFileBaseForFunction}."{started,failed}
+				
+			fi
+			break
```

Comment:
yes, get out of the loop, don't need to create the same samplesheet 6 times (if there is a,b,c,d,e and f)

Reply:
I guess I still don't understand the benefit of merging those samplesheets. If we would have kept them "as is" we would have had smaller projects that would not require brittle `if then else` depending on a specific project name.

Reply:
The same goes for the GS identifiers (e.g. 104832-064), that MUST be in a specific format otherwise the pipeline will not start, it is the same level of brittleness. If I could roll back, then yes i would have gone for the old situation, but I thought that this would be the easier solution since we otherwise have to put the data in the right project folders (beforehand)